docs/markdown/Creating-releases.md

```diff
@@ -51,6 +51,21 @@ for example when done in CI that already does its own testing.
 So with `--no-tests` you can tell Meson "Do not build and test generated
 packages.".
 
+## Modifying release archive contents
+
+By default `meson dist` includes all source code under version control. In
+order to exclude tracked files, include uncommitted files, or modify existing
+files, dist scripts can be used with `meson.add_dist_script`. For example:
+
+```
+meson.add_dist_script('gen_somefile.py')
```

Comment:
IIRC if you use triple backticks followed by "meson" the documentation generator will actually do something fancy here -- we do use that in other places too.

Reply:
good point, done.